packages/tupaia-web/src/pages/ForgotPassword.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { Typography } from '@material-ui/core';
+import styled from 'styled-components';
+import { AuthModalBody, AuthModalButton, Form, RouterButton, TextField } from '../components';
+import { useRequestResetPassword } from '../api/mutations';
+import { FORM_FIELD_VALIDATION, MODAL_ROUTES } from '../constants';
+
+const ModalBody = styled(AuthModalBody)`
+  width: 38rem;
+`;
+const StyledForm = styled(Form)`
+  margin-top: 1rem;
+  width: 22rem;
+  max-width: 100%;
+`;
+
+export const CancelButton = styled(RouterButton).attrs({
+  color: 'default',
+  variant: 'outlined',
+})`
+  text-transform: none;
+  font-size: 1rem;
+  width: 22rem;
+  max-width: 100%;
+  margin-left: 0 !important;
+  padding: 0.375rem 1rem; // to match the height of the primary button
+  border-color: ${({ theme }) => theme.palette.text.secondary};
+  ${AuthModalButton} + & {
+    margin-top: 1.3rem;
+  }
```

Comment:
```suggestion

    margin-top: 1.3rem;
  
```
Since it's only used in one place that style can be simplified